src/vsp/session.cpp

```diff
@@ -300,15 +300,16 @@ list<shared_ptr<session>>& session::get_sessions() {
         if (!file.is_open())
             continue;
 
-        string data;
-        getline(file, data);
-
-        vector<string> data_split = split(data, ':');
-        if (data_split.size() != 4)
+        string host, data;
+        if (!getline(file, host) || !getline(file, data))
             continue;
+        u16 port = stoi(data);
 
-        string host = data_split[0];
-        u16 port = stoi(data_split[1]);
+        size_t count = 0;
+        while (getline(file, data))
```

Comment:
wouldn't it be sufficient to just read the first two lines (host, port) and ignore the others (username, progname)? We dont really need to verify that all information is present, do we?